internal/config/config_test.go

```diff
@@ -36,3 +36,27 @@ func Test_GetConfig(t *testing.T) {
 		})
 	}
 }
+
+func Test_GetConfig_Error(t *testing.T) {
+	tests := []struct {
+		name string
+		args map[string]string
+	}{
+		{
+			name: "Test 1",
+			args: map[string]string{"TMPNOTES_PORT": "6000", "TMPNOTES_REDIS_URL": "rediss://someserver:1234", "TMPNOTES_ENABLE_HSTS": "true", "TMPNOTES_MAX_LENGTH": "2000", "TMPNOTES_UI_MAX_LENGTH": "6000", "TMPNOTES_MAX_EXPIRE": "48"},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			for env, val := range tt.args {
+				os.Setenv(env, val)
+				defer os.Unsetenv(env)
+			}
+			err := GetConfig()
+			if err == nil {
```

Comment:
we should compare for that specific error (e.g. if the error != the specific error)
err != fmt.Errorf("UiMaxLength %v should not be greater than MaxLength %v", Config.UiMaxLength, Config.MaxLength)

Reply:
Good idea, will add another commit for this

Reply:
Added in dbb106794b48139483e5a9a5c560214c3b0fa664